graphql/schema.graphql

```diff
@@ -811,7 +811,7 @@ type Patch {
   patchNumber: Int!
   author: String!
   authorDisplayName: String!
-  version: String!
+  version: Version
```

Comment:
Can you validate that this isn't used on the UI?

Reply:
It's not utilized 👍